io.home-assistant.humidity/manifest.tt

```diff
@@ -0,0 +1,13 @@
+abstract class @io.home-assistant.humidity
+#_[thingpedia_name="Humidity Sensor"]
+#_[thingpedia_description="Interface for Home Assistant's Humidity Sensor."]
+#_[description="Interface for Home Assistant's Humidity Sensor."]
+#[license="CC-0"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(humid,normal), out value : String)
+  #_[canonical="humidity sensor state"]
+  #_[confirmation="the state of your humidity sensor"]
+  #_[formatted=["The humidity sensor says ${state}.", "The humidity sensor says ${value}."]];
```

Comment:
Ok the value of this one is definitely a number.

Reply:
Value is a string because it includes both the number and the unit of measurement e.g. 18°C. Should we do it some other way?

For the devices where there are only binary sensors, actually there is no way of getting a value, so yeah we can actually remove value from these

Reply:
Yeah so please remove value for all things that don't have value. For all other things, we either ignore them for now (no value parameter declared), or we must parse number and unit, then map it to native ThingTalk units (https://almond-dev.stanford.edu/doc/thingtalk-reference.md). We can't use strings to represent numbers, because numbers are preprocessed.

FWIW, I'm sure Home Assistant has a way to get to the raw value, because they themselves support automations and triggers on values.